components/NavBar/DropDownMenu/index.tsx

```diff
@@ -0,0 +1,42 @@
+import NavLink, { NavLinkProps } from '../NavLink'
+import { Transition, Popover } from '@headlessui/react'
+import Hamburger from '@components/Custom/Hamburger'
+
+export interface DropDownMenuProps {
+  links: Array<NavLinkProps>
+}
+
+const DropDownMenu = ({ links }: DropDownMenuProps) => {
+  return (
+    <Popover className='relative'>
+      <Popover.Button className='relative px-4 py-2 font-serif text-2xl text-white hover:opacity-75 focus:outline-none focus:ring focus:ring-teal focus:ring-opacity-50 rounded'>
```

Comment:
not sure why u need the relative class
```suggestion
      <Popover.Button className='px-4 py-2 font-serif text-2xl text-white hover:opacity-75 focus:outline-none focus:ring focus:ring-teal focus:ring-opacity-50 rounded'>
```